.github/workflows/create-codespaces-prebuild.yml

```diff
@@ -0,0 +1,17 @@
+name: Create Codespaces Prebuild
+on:
+  schedule:
+    # Run at 06:00 am UTC every day
+    - cron:  '0 6 * * *'
+  workflow_dispatch:
+jobs:
+  createPrebuild:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: github/codespaces-precache@v1.0.1
+        with:
+          regions: WestUs2
+          sku_name: standardLinux32gb
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What is this for? No token should be needed?

Reply:
See https://github.com/github/codespaces-precache#standard-template

Reply:
Following that to https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input it suggests the default permissions are very broad and one can restrict eg 
```
permissions:
  contents: read
```
But I don't grok how this relates to EXPERIMENTAL_CODESPACE_CACHE_TOKEN.

If you believe we're "doing it right" here then that's fine with me.

Reply:
This is literally copied from the above doc (and I see the exact same in other repos).

https://github.com/github/codespaces-precache#standard-template

```
name: precache codespace
on:
  push:
    branches:
      - main
  workflow_dispatch:
jobs:
  createPrebuild:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: github/codespaces-precache@v1.0.1
        with:
          regions: WestUs2
          sku_name: standardLinux32gb
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

https://github.com/microsoft/vscode/blob/main/.github/workflows/create-codespaces-prebuild.yml

https://github.com/rails/rails/blob/main/.github/workflows/codespaces.yml